server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/routes/MessageIdReindexingRoutes.java

```diff
@@ -50,19 +48,20 @@
 import spark.Service;
 
 @Api(tags = "MessageIdReIndexing")
-@Path("/mailboxIndex")
+@Path("/messages")
```

Comment:
I think you have to modify the `@Path` on line 82, else you are going to introduce a `/messages/messages/{messageId}` endpoint.